src/components/grammar-courses/basic-pronouns-articles/LessonComponent.jsx

```
@@ -0,0 +1,334 @@
+      <div className="bg-dark-blue-4 border-l-4 border-green-500 p-4 rounded-lg">
+        <h3 className="text-lg font-semibold text-green-400 mb-4">
+          <Lightbulb className="inline w-5 h-5 mr-2" />
+          Examples in Context
+        </h3>
+        {lessonData.content.examples.map((category, index) => (
+          <div key={index} className="mb-6">
+            <h4 className="font-medium text-neutral-1 mb-3">{category.category}</h4>
+            <div className="grid gap-2">
+              {category.sentences.map((sentence, idx) => (
+                <div key={idx} className="bg-dark-blue-5 p-3 rounded border border-[#1f2937]">
+                  <p className="text-neutral-2 italic">"{sentence}"</p>
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)